data/wanderer/wanderers start.txt

```diff
@@ -562,12 +528,12 @@ mission "Wanderers: Defend Vara Ke'sok"
 			branch hint
 				has "Wanderers: Defend Vara Ke'sok Hint: done"
 			`Soon after you land in the spaceport, there is a flurry of activity as many of the Wanderers fly to their ships, which begin to take off from the planet. In the sky, you see bright flashes that may be distant weapons fire or explosions.`
-			`	You are buffeted by wind as one of the Wanderers lands next to your ship and says, in the Hai language, "Unfettered [raider, pirate] fleet comes. We have put food on freighter, hope they take [offering, gift] and leave. If not [appeased, satisfied] you help convince them to leave. Yes?"`
+			`	You are buffeted by wind as one of the Wanderers lands next to your ship, speaking so quickly that your translator can barely keep up. "Unfettered [raider, pirate] fleet comes. We have put food on freighter, hope they take [offering, gift] and leave. If not [appeased, satisfied] you help convince them to leave. Yes?"`
```

Comment:
Why did the translator suddenly degrade it's level of translation?

Reply:
The speaker is talking at the very limits of the machine's ability to translate, so it's producing something between "insightful translations" and "meaningless gibberish." This demonstrates that it isn't an on/off thing, but the device does the best it can with the available information.

Reply:
I would love to see more instances of Translator breakdown, to show that this isn't infallible technology.

Reply:
There are several throughout the Wanderers campaign.